Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:8
+
+USER root
+
+WORKDIR /data/cxcloud-process-engine
+
+COPY src/ /data/cxcloud-process-engine/src
+#COPY config/ /data/cxcloud-process-engine/config
```

Comment:
Do we need to leave that commented line?

Reply:
@nadezhda good catch